### PR TITLE
Fastify4

### PR DESCRIPTION
**Pull request by jwilmott**
- Update fastify peer dependency to 4 to work with nestjs 9
- Update circleci node docker image to 14

### CODE REVIEW

**Comment by jwilmott** [comment]
Do I want to add husky back in or ok to leave out?

**Comment by jwilmott** [comment]
Is it ok that package lock version is now 2?

File: package.json [hunk 1]
```diff
@@ -139,32 +139,20 @@
       ]
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
```
Comment (vfrank66) [inline package.json:147]: update but don't remove

Comment (jwilmott) [reply package.json:147]: done

**Pull request merged**